graphql_api/types/repository/repository.graphql

```diff
@@ -79,6 +79,9 @@ type Repository {
     filters: ComponentMeasurementsSetFilters
     orderingDirection: OrderingDirection
   ): [ComponentMeasurements!]!
+  componentsYaml(
+    term_id: String
```

Comment:
let's use `termID` instead of snake case as this deals directly with client

Reply:
updated